vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -28,6 +30,11 @@ def get_pods(inventory)
       process_collection(inventory["pod"], :container_groups) { |n| parse_pod(n) }
     end
 
+    def get_replication_controllers(inventory)
+      process_collection(inventory["replication_controller"], :container_replication_controllers)\
```

Comment:
Please remove the line continuation marker and then only indent 2 spaces on the next line.